### PR TITLE
24). Add topic files 67 to 70

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. It also fixes a few article files from previous commits, and fixes a few redirection entries..

Please merge #17846 and #17892 before merging this.

cc: @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Comment by BillWagner** [comment]
Adding @mairaw, who has been following this.

File: docs/standard/linq/project-new-type.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 48145cf9-1e0b-4e73-bbfd-28fc04800dc4
+---
+
+# How to project a new type (LINQ to XML)
+
+Other examples in this section have shown queries that return results as <xref:System.Collections.Generic.IEnumerable%601> of <xref:System.Xml.Linq.XElement>, <xref:System.Collections.Generic.IEnumerable%601> of `string`, and <xref:System.Collections.Generic.IEnumerable%601> of `int`. These are common result types, but they aren't appropriate for every scenario. In many cases you will want your queries to return an <xref:System.Collections.Generic.IEnumerable%601> of some other type.
```
Comment (mairaw) [inline docs/standard/linq/project-new-type.md:13]:
```suggestion
Other examples in this section have shown queries that return results as <xref:System.Collections.Generic.IEnumerable%601> of <xref:System.Xml.Linq.XElement>, <xref:System.Collections.Generic.IEnumerable%601> of `string`, and <xref:System.Collections.Generic.IEnumerable%601> of `int`. These are common result types, but they aren't appropriate for every scenario. In many cases, you'll want your queries to return an <xref:System.Collections.Generic.IEnumerable%601> of some other type.
```

File: docs/standard/linq/project-new-type.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                n.<ProductName>.Value, CInt(n.<Quantity>.Value))
+
+        For Each n As NameQty In nqList
+            Console.WriteLine(n.name & ":" & n.qty)
+        Next
+    End Sub
+End Class
+```
+
+This example uses the <xref:System.Xml.Linq.XContainer.Element%2A> method that was introduced in the article [How to retrieve a single child element](retrieve-single-child-element.md). It also uses casts to retrieve the values of the elements that are returned by the <xref:System.Xml.Linq.XContainer.Element%2A> method.
```
Comment (mairaw) [inline docs/standard/linq/project-new-type.md:75]:
```suggestion
This example uses the <xref:System.Xml.Linq.XContainer.Element%2A> method that was introduced in the [How to retrieve a single child element](retrieve-single-child-element.md) article. It also uses casts to retrieve the values of the elements that are returned by the <xref:System.Xml.Linq.XContainer.Element%2A> method.
```

File: docs/standard/linq/project-new-type.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+The example produces this output:
```
Comment (mairaw) [inline docs/standard/linq/project-new-type.md:77]:
```suggestion
The example produces the following output:
```

File: docs/standard/linq/query-linq-xml-xpath.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+title: How to query LINQ to XML using XPath - LINQ to XML
+description:
```
Comment (mairaw) [inline docs/standard/linq/query-linq-xml-xpath.md:3]: missing description

File: docs/standard/linq/query-linq-xml-xpath.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  - "csharp"
+  - "vb"
+ms.assetid: ee5af263-4ab1-45e5-b792-33a3221b426d
+---
+
+# How to query LINQ to XML using XPath (LINQ to XML)
+
+This article introduces the extension methods that enable you to query an XML tree by using XPath. For detailed information about using these extension methods, see <xref:System.Xml.XPath.Extensions?displayProperty=nameWithType>.
+
+> [!NOTE] Unless you have a very specific reason for querying using XPath, such as extensive use of legacy code, using XPath with LINQ to XML isn't recommended. XPath queries won't perform as well as LINQ to XML queries.
```
Comment (mairaw) [inline docs/standard/linq/query-linq-xml-xpath.md:15]:
```suggestion
> [!NOTE] 
> Unless you have a very specific reason for querying using XPath, such as extensive use of legacy code, using XPath with LINQ to XML isn't recommended. XPath queries won't perform as well as LINQ to XML queries.
```

File: docs/standard/linq/query-linq-xml-xpath.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+## Example
+
+ The following example creates a small XML tree and uses <xref:System.Xml.XPath.Extensions.XPathSelectElements%2A> to select a set of elements.
```
Comment (mairaw) [inline docs/standard/linq/query-linq-xml-xpath.md:19]:
```suggestion
The following example creates a small XML tree and uses <xref:System.Xml.XPath.Extensions.XPathSelectElements%2A> to select a set of elements.
```

File: docs/standard/linq/write-linq-xml-axis-method.md [hunk 7]
```diff
@@ -0,0 +1,477 @@
+---
+title: How to write a LINQ to XML axis method - LINQ to XML
+description: You can write your own axis methods to retrieve collections from an XML tree, methods that meet the needs of your applications. Writing extension methods is one of the best ways to do this. The article provides an example for C# and Visual Basic.
```
Comment (mairaw) [inline docs/standard/linq/write-linq-xml-axis-method.md:3]:
```suggestion
description: Learn how to write your own axis methods to retrieve collections from an XML tree in C# or Visual Basic.
```

**Review by mairaw — COMMENTED** [review]
LGTM. Left some comments for you to review before merging. It seems some of the files were also part of the other PRs?

**Comment by mairaw** [comment]
Ping me when changes are applied and conflicts resolved @jameshkramer. Thanks!

File: .openpublishing.redirection.json [hunk 8]
```diff
@@ -75,7 +75,7 @@
     },
     {
       "source_path": "docs/architecture/modernize-with-azure-containers/lift-and-shift-existing-apps-devops/when-to-deploy-windows-containers-to-azure-container-service-kubernetes.md",
-      "redirect_url": "/dotnet/architecture/modernize-with-azure-containers//modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes"
+      "redirect_url": "/dotnet/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes"
```
[the remark this answers is not in the document]
Comment (jameshkramer) [reply .openpublishing.redirection.json:78]: Yes, whenever I have been encountering errors in redirection I have been fixing them. Why are you breaking them? I understand that you decided to fix them in master right away, but what is the harm if we fix them in linq-to-xml?

File: docs/standard/linq/project-new-type.md (hunk 3 above)
Comment (jameshkramer) [reply docs/standard/linq/project-new-type.md]: I will submit a PR this afternoon which fixes this everywhere, including in PRs that are not yet merged.

File: docs/standard/linq/query-linq-xml-xpath.md (hunk 4 above)
Comment (jameshkramer) [reply docs/standard/linq/query-linq-xml-xpath.md]: I added a description.

**Comment by jameshkramer** [comment]
@mairaw @BillWagner 

Maira, I made the changes. There is still an unresolved conversation where I added a description.

**Review by mairaw — APPROVED** [review]

**Pull request merged**